Automattic/vip-cli
# chore(deps): Update Babel-related packages

## Description

⚠️ needs to go after #1153 to avoid merge conflicts.

  * Bump @babel/cli from 7.15.7 to 7.19.3;
  * Bump @babel/core from 7.14.0 to 7.19.6;
  * Bump @babel/eslint-parser from 7.13.14 to 7.19.1;
  * Bump @babel/plugin-transform-modules-commonjs from 7.14.0 to 7.19.6;
  * Bump @babel/preset-env from 7.14.1 to 7.19.4;
  * Bump @babel/preset-flow from 7.13.13 to 7.18.6.

## Steps to Test

1. Check out the PR.
2. Run `npm run build`. It should succeed.
